lib/screens/filters.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:meals_app/providers/filters_provider.dart';
+
+/// Screen that contains the filters.
+/// Here you can set the filters for which meals are shown.
+/// Is ConsumerWidget because it uses the [filterProvider].
+class FiltersScreen extends ConsumerWidget {
+  const FiltersScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final filters = ref.watch(filterProvider);
+
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Filters"),
+      ),
+      /* drawer: MainDrawer(
+        onSelectScreen: (identifier) {
+          Navigator.of(context).pop();
+          if (identifier == "meals") {
+            Navigator.of(context).pushReplacement(
+                MaterialPageRoute(builder: (ctx) => const TabsScreen()));
+          }
+        },
+      ), */
+      body: Column(
```

Comment:
You could have extracted SwitchListTiles here into your own widget that take what is different  between them as parameters. This would reduce code duplication